test/e2e/steps/given.ts

```diff
@@ -51,7 +51,8 @@ const basicConfig = {
   el: '#okta-login-container',
   authParams: { pkce:false },
   clientId: WIDGET_WEB_CLIENT_ID,
-  scopes: ['openid', 'email', 'profile']
```

Comment:
nit: rename `basicConfig` to be something specific about authn flow, as it's not the default one anymore